src/ports/postgres/modules/knn/knn.py_in

```diff
@@ -88,12 +88,28 @@ def knn_validate_src(schema_madlib, point_source, point_column_name, point_id,
         plpy.error("kNN Error: Data type '{0}' is not a valid type for"
                    " column '{1}' in table '{2}'.".
                    format(col_type_test, test_id, test_source))
+
+    fn_dist = fn_dist.lower().strip()
```

Comment:
What's the expected behavior if `fn_dist = None` (i.e. user inputs `NULL` for the `fn_dist`)?

Reply:
I have added a check here. If fn_dist is None, set it to default 'squared_dist_norm2' function.